src/hooks/index.js

```diff
@@ -202,7 +202,15 @@ exports.restrictToCurrentSchool = hook => {
 		});
 		if (access)
 			return hook;
-		hook.params.query.schoolId = res.data[0].schoolId;
```

Comment:
was genau soll das hier werden?
restrictToCurrentSchool hängt den Query Param dran, was ihr jetzt macht ist einfach undefined mit einem Wert vergleichen ...
Im Frontend wird die schoolId net immer mitgeschickt, das regelt in dem Fall der Hook, aber den macht ihr durch Zeile 205-213 unbrauchbar

Reply:
und bitte nochmal bitte spacing beachten...

Reply:
Das stimmt leider wirklich, wir haben durch das Löschen dieser Zeile keine Garantie, dass im query eine `schoolId` vorhanden ist, so dass im Frontend (vermutlich) eine Menge kaputt gehen wird. Ich weiß jetzt schon, dass z.B. bei den CourseGroups keine mitgeschickt wird, bei den Kursen vermutlich auch nich mehr, seitdem wir diesen Hook hatten.

Reply:
hmm, dann würde ich gerne zwei Hooks machen, einen der eine fehlende ID ergänzt und einen der ein Fehler schmeisst wenn man eine falsche ID schickt ... oder meinetwegen in beiden, aber das automatsiche Gradeziehen bei einer falschen ID gefällt mir nicht.. ich mach mal ein Vorschlag..

Reply:
@Langleu BItte neue Version nochmal checken